vignettes/admiral.Rmd

```diff
@@ -163,120 +97,157 @@ and `TRTSTMF` variables.
+    parameters = c("SYSBP", "DIABP"),
+    set_values_to = exprs(
+      AVAL = (AVAL.SYSBP + 2 * AVAL.DIABP) / 3,
+      PARAMCD = "MAP",
+      PARAM = "Mean Arterial Pressure (mmHg)",
+      AVALU = "mmHg"
+    )
+  )
+```
 
-- For arguments which expect one or more variable names, a list of symbols is
-expected, e.g. `by_vars = exprs(PARAMCD, AVISIT)` 
+```{r, eval=TRUE, echo=FALSE}
+advs %>%
+  arrange(USUBJID, AVISITN, PARAMCD) %>%
+  dataset_vignette(
+    display_vars = exprs(USUBJID, AVISIT, PARAMCD, AVAL)
+  )
+```
 
-- For arguments which expect a single expression, the expression needs to be
-passed "as is", e.g. `filter = PARAMCD == "TEMP"`
+Note: For the users' convenience, `{admiral}` actually provides
+`derive_param_map()`  (wrapper of `derive_param_computed()`) to derive MAP.
+The above example serves for illustrative purposes only.
 
-- For arguments which expect one or more expressions, a list of expressions is
-expected, e.g. `order = exprs(AVISIT, desc(AESEV))`
+# Other Types of Functions
 
-## What is `exprs()`?
+Along with the derivation functions, `{admiral}` provides a large arsenal
+of functions to help you with your ADaM derivations. Here are some of the 
+other categories.
 
-`exprs()` is a function from the `{rlang}` package. It is used to create a list
-of expressions. The expressions are not evaluated. Rather they are passed on to
-the derivation function which evaluates them in its own environment. This allows
-the derivation function to evaluate the expressions in the context of the input
-dataset.
-Specifically, the `exprs()` function allows users to pass variable names of datasets 
-to the function without wrapping them in quotation marks. 
+## Higher Order Functions
 
-# Handling of Missing Values
+[Higher order functions](../reference/index.html#higher-order) 
+are `{admiral}` functions that take other functions as input. 
+They enhance the existing portfolio of derivation functions by allowing greater 
+customization of the latter's behavior. This is done by allowing a derivation
+function to be:
 
-When using the `{haven}` package to read SAS datasets into R, SAS-style character missing values, i.e. `""`, are *not* converted into proper R `NA` values. Rather they are kept as is. This is problematic for any downstream data processing as R handles `""` just as any other string. Thus, before any data manipulation is being performed SAS blanks should be converted to R `NA`s using `{admiral}`'s `convert_blanks_to_na()` function, e.g.
+* Called multiple times, while varying some of the input arguments (`call_derivation()`);
+* Executed on a subset of the input dataset (`restrict_derivation()`);
+* Executed differently on subsets of the input dataset (`slice_derivation()`).
 
-```r
-dm <- haven::read_sas("dm.sas7bdat") %>% 
-  convert_blanks_to_na()
-```
+Higher order functions are a relatively advanced topic within `{admiral}`; you can
+can read all about them in their [dedicated vignette](higher_order.html).
```

Comment:
SHould we just say the name of the vignette?

Reply:
changed